wallet/wallet.go

```diff
@@ -225,7 +216,8 @@ func newWallet(vb uint16, esm bool, btm dcrutil.Amount, addressReuse bool,
 		BalanceToMaintain:        btm,
 		CurrentStakeDiff:         &StakeDifficultyInfo{nil, -1, -1},
 		lockedOutpoints:          map[wire.OutPoint]struct{}{},
-		feeIncrement:             feeIncrement,
+		feeIncrementMainnet:      FeeIncrementMainnet,
```

Comment:
You should just declare a single fee increment here, since mainnet and testnet will never be enabled at the same time. Put the switch case above the struct declaration.